biomolecules/geom/write_geom_drugs_structures.py

```diff
@@ -0,0 +1,92 @@
+import os
+import pickle
+import argparse
+import tqdm
+
+from rdkit.Chem.rdmolfiles import MolToXYZFile
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--base_path",
+        type=str,
+        required=True,
+        help="Path to where you untarred the rdkit folder i.e. path/to/rdkit_folder",
+    )
+    parser.add_argument(
+        "--parent_path",
+        type=str,
+        required=True,
+        help="Path to the parent directory where structures will be written",
+    )
+    parser.add_argument(
+        "--geom_ids_path",
+        type=str,
+        required=True,
+        help="Path to geom ids pkl to write structures for",
+    )
+    parser.add_argument(
+        "--idmol_path",
+        type=str,
+        required=True,
+        help="Path to id_mol_dict.pkl",
+    )
+    parser.add_argument(
+        "--start_idx",
+        type=int,
+        required=True,
+        help="Index in split ids to start writting files",
+    )
+    parser.add_argument(
+        "--stop_idx",
+        type=int,
+        required=True,
+        help="Index in split ids to stop writting files",
+    )
+
+    return parser.parse_args()
+
+
+def main():
+
+    args = parse_args()
+
+    # rdkit folder downloaded 2024-03-01
+    base_path = args.base_path
+    parent_path = args.parent_path
+    geom_ids_path = args.geom_ids_path
+    idmol_path = args.idmol_path
+    start_idx = args.start_idx
+    stop_idx = args.stop_idx
+
+    # read split ids
+    with open(geom_ids_path, "rb") as f:
+        geom_ids = pickle.load(f)
+
+    with open(idmol_path, "rb") as f:
+        idmol = pickle.load(f)
+
+    for i in tqdm.tqdm(geom_ids[start_idx:stop_idx]):
```

Comment:
minor quibble: maybe "id" instead of "i" to make it clear that this variable is not a simple counter

Reply:
fixed